lib/Site.js

```diff
@@ -99,9 +98,8 @@ const GENERATE_SITE_LOGGING_KEY = 'Generate Site';
 const MARKBIND_WEBSITE_URL = 'https://markbind.github.io/markbind/';
 const MARKBIND_LINK_HTML = `<a href='${MARKBIND_WEBSITE_URL}'>MarkBind ${CLI_VERSION}</a>`;
 
-function Site(rootPath, outputPath, forceReload = false) {
-  const configPath = findUp.sync(SITE_CONFIG_NAME, { cwd: rootPath });
```

Comment:
Removing this line of code means that the user can no longer serve inside the subdirectories, but have to always go back to the root directory and serve. Not sure whether it is a good idea since it can be a convenience feature for the user.

Reply:
Hmm alright, then it might be better to enforce that all alternate site.json files be in the root folder. I don't really want to mess with the config path logic in case something messes up